docs/buildah-build.1.md

```diff
@@ -798,6 +799,8 @@ buildah build --dns-search=example.com --dns=223.5.5.5 --dns-option=use-vc .
 
 buildah build -f Containerfile.in -t imageName .
 
+buildah build --network mynet .
+
```

Comment:
We'll eventually need similar changes to the man page for `podman build` if not already there.